.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
+    "env": {
+      "browser": true,
+      "jest": true,
+      "node": true
+    },
+    "settings": {
+      "react": {
+        "pragma": "React",
+        "version": "^16.4.2"
+      }
+    },
+    "parser": "babel-eslint",
+    "plugins": [
+      "react"
+    ],
+    "rules": {
+      "func-names": ["error", "never"],
+      "semi": ["error", "always"],
+      "quotes": ["error", "single"],
+      "camelcase": [ 1 ],
+      "no-undef": [ 1 ],
+      "react/jsx-curly-spacing": [ 2, "always" ],
+      "react/jsx-indent": [ 2, 4 ],
+      "react/prop-types": [ 1 ]
```

Comment:
We should a popular style guide like `airbnb` and not write rules of our own. This will force us to kill our bad practices.

Reply:
These are rules for react.

Reply:
Why you added rules for `func-names`, `semi`, `quotes`, `camelcase`, `no-undef`?